substrate/frame/revive/src/vm/evm/instructions/control.rs

```diff
@@ -126,7 +126,6 @@ pub fn ret<'ext, E: Ext>(context: Context<'_, 'ext, E>) {
 
 /// EIP-140: REVERT instruction
 pub fn revert<'ext, E: Ext>(context: Context<'_, 'ext, E>) {
-	check!(context.interpreter, BYZANTIUM);
```

Comment:
do we want to remove spec checks in all of our opcodes or it could be useful for testing? (e.g. running tests that assume Byzantium semantics vs Istanbul).

Reply:
we will only run tests for the latest spec, you can specify what fork to target in the EF tests framework